chromium_presubmit_config.json5

```diff
@@ -299,6 +299,9 @@
       "components/cosmetic_filters/renderer/cosmetic_filters_js_handler\\.h",
       "components/speedreader/renderer/speedreader_js_handler\\.h",
     ],
+    "CheckStableMojomChanges": [
+      "components/services/bat_ads/public/interfaces/bat_ads\\.mojom",
```

Comment:
please add a comment:
`# A temporary fix to silence a crashing presubmit check on a deleted file.`